docs/slis/jupyterhub.md

```diff
@@ -0,0 +1,417 @@
+## Goal
+
+Identify key SLIs for the JupyterHub service and document them in: [https://gitlab.cee.redhat.com/data-hub/dh-runbooks/blob/master/JUPYTERHUB.md](https://gitlab.cee.redhat.com/data-hub/dh-runbooks/blob/master/JUPYTERHUB.md)
```

Comment:
I would omit this file for now until we find a place for non argo-cd SLIs
And it links to an internal only runbook :)